master/internal/webhooks/postgres_webhook.go

```diff
@@ -272,6 +280,121 @@ func (l *WebhookManager) deleteWebhook(ctx context.Context, id WebhookID) error
 	return nil
 }
 
+func handleCustomTriggerData(ctx context.Context, data CustomTriggerData, experimentID int, trialID *int) error {
+	var m struct {
+		bun.BaseModel `bun:"table:experiments"`
+		model.Experiment
+		ConfigBytes []byte `bun:"config"`
+	}
+	err := db.Bun().NewSelect().Model(&m).ExcludeColumn("username").Where("id = ?", experimentID).Scan(ctx)
+	if err != nil {
+		return fmt.Errorf("error getting experiment from id %d: %w", experimentID, err)
+	}
+
+	activeConfig, err := expconf.ParseAnyExperimentConfigYAML(m.ConfigBytes)
+	if err != nil {
+		return fmt.Errorf("error parsing experiment config: %w", err)
+	}
+	integrationsConfig := activeConfig.Integrations()
+	if integrationsConfig == nil {
+		return nil
+	}
+	webhookConfig := integrationsConfig.Webhooks
+	if webhookConfig == nil || webhookConfig.Exclude {
+		return nil
+	}
+
+	workspaceID, err := experiment.GetWorkspaceFromExperiment(ctx, &m.Experiment)
+	if err != nil {
+		return fmt.Errorf("error getting workspace from experiment : %w", err)
+	}
+
+	var es []Event
+	if webhookConfig.WebhookID != nil {
```

Comment:
(not a blocker)From the user perspective, where can a user find out the id of a webhook?

Reply:
Since webhook is not part of the CLI, user can only access the webhook ids from REST api. This is alright to me because we want to encourage user to use webhook name for readability, and using id is more like an add-on.